src/integrations/zhipu_glm.py

```
@@ -0,0 +1,157 @@
+            "temperature": options.get("temperature", 0.7),
+            "max_tokens": options.get("max_tokens", 1024),
+            "stream": False
+        }
+
+        try:
+            async with self.session.post(
+                self.config.base_url,
+                json=payload
+            ) as response:
+                if response.status != 200:
+                    raise Exception(f"GLM API returned status {response.status}: {await response.text()}")
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)